apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -1402,6 +1402,7 @@ where
             let unshield = wrapper
                 .check_and_generate_fee_unshielding(
                     transfer_code_hash,
+                    Some("tx_transfer.wasm".to_string()),
```

Comment:
this should be a constant?

Reply:
Fixed this and defined these WASM paths in one place. Thanks!